controllers/storagecluster/provider_server.go

```diff
@@ -55,6 +56,10 @@ func (o *ocsProviderServer) ensureCreated(r *StorageClusterReconciler, instance
 		return res, nil
 	}
 
+	if err := o.createNodePortService(r, instance); err != nil {
```

Comment:
@iamniting Can we add a comment above the call site to indicate that we are keeping this to enable the transition of node port based connectivity to load balancer based connectivity during 4.11 lifetime. And that this is going to be deprecated and removed in a future version.

Reply:
done